README.md

```diff
@@ -172,82 +100,92 @@ In Device Simulator Express, you can use keyboard to interact with the device:
+
+### I. Take a look at the "Device Simulator Express: Getting Started" Command.
+1. Type in `"Device Simulator Express: Getting Started"` in the command palette (`CTRL + SHIFT + P` / `CMD + SHIFT + P` to open the command palette).  
 <img alt='Getting Started' src='https://raw.githubusercontent.com/microsoft/vscode-python-devicesimulator/dev/assets/readmeFiles/getting_started.png'>
-2. Choose the `micro:bit` option from the dropdown.
-3. Read, copy and learn some of the things you can do with the simulator!
-   
-#### II. Start with the "Device Simulator Express [micro:bit]: New File" Command.
+1. Choose the the device you want to play with from the dropdown.
+2. Read, copy and learn some of the things you can do with the simulator!
+
+### II. Start with the "Device Simulator Express: New File" Command.
 
-1. Type in `"Device Simulator Express: [micro:bit] New File"` in the command palette (`CTRL + SHFT + P / CMD + SHIFT + P` to open the command palette).  
-   <img alt='"New micro:bit File" animation' src='https://github.com/microsoft/vscode-python-devicesimulator/blob/dev/assets/readmeFiles/microbit/microbit-new-file.gif'>
-2. Name and save your file somewhere, and we’re good to go!
+<img alt='"New File" animation' src='assets/readmeFiles/new_file.gif'>
 
-#### III. Start from an existing Python file.
+1. Type in `"Device Simulator Express: New File"` in the command palette (`CTRL + SHIFT + P` / `CMD + SHIFT + P` to open the command palette).  
+2. Select the device you want to use.
+3. Name and save your file somewhere, and we’re good to go!
+4. Start with some examples: you can find examples files and tutorials inside the comments at the top of the file.
+
+### III. Start from an existing Python file.
 
 1. Open the folder or your .py file in Visual Studio Code.
-2. Run `Device Simulator Express: [micro:bit] Open Simulator` from the command palette or icon in the editor toolbar.
+2. Run `Device Simulator Express: Open Simulator` from the command palette or icon in the editor toolbar.
+3. Select the device you want to use.
 
-#### IV. Run your code on the simulator.
+### IV. Run your code on the simulator.
 
-<img alt='How to run the micro:bit simulator animation' src='https://raw.githubusercontent.com/microsoft/vscode-python-devicesimulator/dev/assets/readmeFiles/microbit/microbit-run.gif'>
+<img alt='How to run the simulator animation' src='assets/readmeFiles/run.gif'>
 
 1. Run `Run Simulator` from the command palette or use the `Play` button on the simulator webview.
 
-#### V. Deploy your code to the physical device
+### V. Deploy your code to the physical device
 
-1. Run `[micro:bit] Deploy to Device` from the command palette
+Before deploying the Python code to your CPX device, you need to format your device by following these tutorials:
 
-<img alt='Deploy to micro:bit device' src='https://github.com/microsoft/vscode-python-devicesimulator/blob/dev/assets/readmeFiles/microbit/microbit-deploy.png'>
+*For the CPX*:

```

Comment:
This didn't format well in markdown for some reason (by checking it out at https://github.com/microsoft/vscode-python-devicesimulator/tree/users/t-vali/command-p-restructure). You can add CPX and micro:bit as a bullet topic each? Or maybe something like...
> Before deploying the Python code to your CPX device, you need to format your device by following these tutorials:
>- CPX
> a. Step one;
> b. Step two...
>- micro:bit
> a. Step one...

Reply:
I believe I fixed the formatting with all of them. I wanted to use a. b. or 1.1. 1.2.  but the way standard markdown works, it turns them into code blocks or doesn't indent them properly.